wazuh/wazuh-dashboard-plugins
# Support for CDB lists beginning with quotes

### Description

Fixes the rendering of the table as in some cases they were not rendering correctly.

For example: 
- “:key” was not rendering in the table.

And added validations in the inputs.

### Issues Resolved
- #7166

### Evidence

![image](https://github.com/user-attachments/assets/55f9ae69-493a-4d22-9576-c43204bba77d)

<img width="1298" alt="image" src="https://github.com/user-attachments/assets/c4bed83f-dda2-4031-8c87-1cff398d046a">

### Test

1. Navigate to CDB list
2. Create CDB list with different keys
   - `key:`
   - `"key":key`
   - `"key:":"key:"`
   - `key1:"key:" `
3. The table should be rendered accordingly.
4. The keys and values in quotes must be at the beginning and at the end, not in the middle of the value.  

### Check List
